src/util/chaintype.cpp

```diff
@@ -41,3 +41,20 @@ std::optional<ChainType> ChainTypeFromString(std::string_view chain)
         return std::nullopt;
     }
 }
+
+std::string ChainTypeToDisplayString(ChainType chain)
```

Comment:
The existing `ChainTypeToString()` in the same file above seems to be already invoked for displaying user-facing messages. Perhaps simpler to use it rather than adding this new one.

Reply:
Or maybe only store the difference:
```C++
std::string ChainTypeToDisplayString(ChainType chain)
{
    switch (chain) {
    case ChainType::MAIN:
        return "Bitcoin";
    case ChainType::TESTNET:
        return "testnet";
    case ChainType::TESTNET4:
    case ChainType::SIGNET:
    case ChainType::REGTEST:
        return ChainTypeToString(chain);
    }
    assert(false);
}
```

Reply:
It would be good to get a response here why there is a new method instead of reusing the existing.